app/static/js/supervisorPortal.js

```diff
@@ -9,6 +9,33 @@ $(document).ready(function(){
   $('#formSearchButton').on('click', function(){
     runFormSearchQuery(newData='', false);
   });
+  $('#addUserToDept').on('click', function() {
+      $("#addSupervisorToDeptModal").modal("show");
+  })
+
+  $('#addUser').on('click', function() {
+      let supervisor = $('#supervisorModalSelect :selected').val()
+      let department = $('#departmentModalSelect :selected').val()
+      let data = {"supervisor": supervisor, "department": department}
+      $.ajax({
+        method: "POST",
+        url: "/supervisorPortal/addUserToDept",
+        data: data,
+        success: function(response) {
+          if (response == "True") {
+            $("#flash_container").html('<div class="alert alert-success" role="alert" id="flasher">Supervisor has been added to department.</div>');
```

Comment:
Why are you using these flashers manually instead of using the `msgFlash` function?